### ISSUE TITLE
Failed update removed conda command

### ISSUE THREAD

**Issue by ikivimak** [conda/conda#2249]
I'm on OSX 10.11 with Anaconda 2.4.1. I wanted to install pandoc with conda, because of which conda planned to update itself:

```
$ conda install pandoc
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ....
Solving package specifications: ...............
Package plan for installation in environment //anaconda:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    openssl-1.0.2g             |                0         3.0 MB
    pandoc-1.15.2              |                0        25.0 MB
    setuptools-20.2.2          |           py27_0         453 KB
    conda-4.0.4                |           py27_0         184 KB
    pip-8.1.0                  |           py27_0         1.5 MB
    ------------------------------------------------------------
                                           Total:        30.1 MB

The following NEW packages will be INSTALLED:

    pandoc:     1.15.2-0     

The following packages will be UPDATED:

    conda:      3.19.3-py27_0 --> 4.0.4-py27_0 
    openssl:    1.0.2f-0      --> 1.0.2g-0     
    pip:        8.0.3-py27_0  --> 8.1.0-py27_0 
    setuptools: 20.1.1-py27_0 --> 20.2.2-py27_0

Proceed ([y]/n)? 
```

Everything went well until the conda update failed because of a permission error:

```
Fetching packages ...
openssl-1.0.2g 100% |######################################################| Time: 0:00:03   1.05 MB/s
pandoc-1.15.2- 100% |######################################################| Time: 0:00:14   1.75 MB/s
setuptools-20. 100% |######################################################| Time: 0:00:00 658.04 kB/s
conda-4.0.4-py 100% |######################################################| Time: 0:00:00 292.35 kB/s
pip-8.1.0-py27 100% |######################################################| Time: 0:00:03 507.07 kB/s
Extracting packages ...
[      COMPLETE      ]|#########################################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|#########################################################################| 100%
Linking packages ...
An unexpected error has occurred, please consider sending the                                   |   0%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "//anaconda/bin/conda", line 5, in <module>

  File "//anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 195, in main
    args_func(args, p)
  File "//anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 202, in args_func
    args.func(args, p)
  File "//anaconda/lib/python2.7/site-packages/conda/cli/main_install.py", line 45, in execute
    install.install(args, parser, 'install')
  File "//anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 423, in install
    plan.execute_actions(actions, index, verbose=not args.quiet)
  File "//anaconda/lib/python2.7/site-packages/conda/plan.py", line 537, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "//anaconda/lib/python2.7/site-packages/conda/instructions.py", line 148, in execute_instructions
    cmd(state, arg)
  File "//anaconda/lib/python2.7/site-packages/conda/instructions.py", line 91, in LINK_CMD
    link(state['prefix'], arg, index=state['index'])
  File "//anaconda/lib/python2.7/site-packages/conda/instructions.py", line 87, in link
    install.link(pkgs_dir, prefix, dist, lt, index=index)
  File "//anaconda/lib/python2.7/site-packages/conda/install.py", line 645, in link
    update_prefix(join(prefix, f), prefix, placeholder, mode)
  File "//anaconda/lib/python2.7/site-packages/conda/install.py", line 291, in update_prefix
    os.remove(path) # Remove file before rewriting to avoid destroying hard-linked cache.
OSError: [Errno 13] Permission denied: '/anaconda/include/openssl/opensslconf.h'
```

As a result, the conda command was removed:

```
$ conda
-bash: //anaconda/bin/conda: No such file or directory
```

How can I get it back?

**Comment by jakirkham** [comment]
What are the permissions on `/anaconda`?

**Comment by ikivimak** [comment]
They're "drwxr-xr-x"

**Comment by jakirkham** [comment]
Owned by?

**Comment by ikivimak** [comment]
It's owned by my user, which has admin rights. 

Hmmm, but now that I think a bit more, maybe this is caused by the fact that when I started using the computer, I had some confusion with setting up the accounts. I don't remember what I did exactly, but I think that I first set up a new account and installed Anaconda under that. But later I decided to migrate my old user from my older Mac using Time Machine. After that I removed the "new" account and have been using the one from the old Mac. And it might be that both users had the same username, which, I now realize, is not the most optimal situation...

But again, I'm not completely sure what I did. It was a while ago and I tested several things, as I couldn't make up my mind on whether to migrate the old account or start using a completely new one...
